lib/central_mail/datestamp_pdf.rb

```diff
@@ -13,18 +13,23 @@ def initialize(file_path, append_to_stamp: nil)
     def run(settings)
       stamp_path = Common::FileHelpers.random_file_path
       generate_stamp(stamp_path, settings[:text], settings[:x], settings[:y], settings[:text_only], settings[:size],
-                     settings[:timestamp], settings[:page_number], settings[:template])
+                     settings[:timestamp], settings[:page_number], settings[:template], @file_path)
       stamp(@file_path, stamp_path, multistamp: settings[:multistamp])
     ensure
       Common::FileHelpers.delete_file_if_exists(stamp_path) if defined?(stamp_path)
     end
 
     # rubocop:disable Metrics/ParameterLists
     # rubocop:disable Metrics/MethodLength
-    def generate_stamp(stamp_path, text, x, y, text_only, size = 10, timestamp = nil, page_number = nil, template = nil)
+    def generate_stamp(stamp_path, text, x, y, text_only, size = 10, timestamp = nil, page_number = nil,
+                       template = nil, file_path = nil)
       timestamp ||= Time.zone.now
       unless text_only
-        text += " #{I18n.l(timestamp, format: :pdf_stamp)}"
+        text += if file_path == 'tmp/vba_40_10007-stamped.pdf'
```

Comment:
Are there any specs that you can update to support this change?

Reply:
Ok I added two test cases to `spec/lib/central_mail/datestamp_pdf_spec.rb` for this